stdlib/ballerina-builtin/src/main/ballerina/ballerina/io/natives.bal

```diff
@@ -18,28 +18,28 @@ public struct TextRecordChannel{
 @Return {value:"CharacterChannel converted from ByteChannel"}
 public native function <ByteChannel channel> toCharacterChannel(string encoding)(CharacterChannel);
 
-@Description {value:"Function to convert a CharacterChannel to TextRecordChannel"}
+@Description {value:"Function to convert a CharacterChannel to DelimitedRecordChannel"}
 @Param {value:"channel: The CharacterChannel to be converted"}
 @Param {value:"recordSeparator: Terminating expression to distinguish between records"}
 @Param {value:"fieldSeparator: Terminating expression to distinguish between fields"}
-@Return {value:"TextRecordChannel converted from CharacterChannel"}
+@Return {value:"DelimitedRecordChannel converted from CharacterChannel"}
 public native function <CharacterChannel channel> toTextRecordChannel(string recordSeparator,
                                                                       string fieldSeparator)
-(TextRecordChannel);
+(DelimitedRecordChannel);
 
 @Description {value:"Function to read text records"}
 @Param {value:"channel: The TextRecordChannel to read text records from"}
 @Return {value:"Fields listed in the record"}
-public native function <TextRecordChannel channel> nextTextRecord()(string []);
+public native function <DelimitedRecordChannel channel> nextTextRecord () (string[]);
 
 @Description {value:"Function to write text records"}
-@Param {value:"channel: The TextRecordChannel to write text records to"}
+@Param {value:"channel: The DelimitedRecordChannel to write records"}
 @Param {value:"records: Fields which are included in the record"}
-public native function <TextRecordChannel channel> writeTextRecord(string [] records);
+public native function <DelimitedRecordChannel channel> writeTextRecord (string[] records);
 
 @Description{value:"Function to close the text record channel"}
 @Param {value:"channel: The TextRecordChannel to be closed"}
```

Comment:
Param value needs to change to DelimitedRecordChannel from TextRecordChannel.

Reply:
c65b5580d110f274fb2a630c136996a31c17d89e